### DISCUSSION TITLE
Why is the schema name case different in the drizzle adapter?

### DISCUSSION THREAD

**Discussion by myodan** [nextauthjs/next-auth#11325] [Help] [answered]
The example below is the schema used by the basic drizzle adapter described in the documentation.

```typescript
export const users = pgTable("user", {
  id: text("id")
    .primaryKey()
    .$defaultFn(() => crypto.randomUUID()),
  name: text("name"),
  email: text("email").notNull(),
  emailVerified: timestamp("emailVerified", { mode: "date" }), // camelCase
  image: text("image"),
})
 
export const accounts = pgTable(
  "account",
  {
    userId: text("userId") // camelCase
      .notNull()
      .references(() => users.id, { onDelete: "cascade" }),
    type: text("type").$type<AdapterAccountType>().notNull(),
    provider: text("provider").notNull(),
    providerAccountId: text("providerAccountId").notNull(),    // camelCase
    refresh_token: text("refresh_token"),                      // snake-case
    access_token: text("access_token"),                        // snake-case
    expires_at: integer("expires_at"),                         // snake-case
    token_type: text("token_type"),                            // snake-case
    scope: text("scope"),
    id_token: text("id_token"),                                // snake-case
    session_state: text("session_state"),                      // snake-case
  },
  (account) => ({
    compoundKey: primaryKey({
      columns: [account.provider, account.providerAccountId],
    }),
  })
)
```
If you look at the comments in the code above, you'll see that the schema names have different cases. Why is this done this way? Is there a special reason? I'm curious.

If you change it arbitrarily, Typescript will complain, which is understandable because it references it internally against its schema.

Is there any way to change this?

**Comment by k-taro56** [comment, marked as answer]
> Auth.js uses camelCase for its database rows while respecting the conventional snake_case formatting for OAuth-related values. If the mixed casing is an issue for you, most adapters have a dedicated documentation section on how to force a casing convention.

https://authjs.dev/concepts/database-models

**Reply by myodan** [reply]
> Auth.js uses camelCase for its database rows while respecting the conventional snake_case formatting for OAuth-related values. If the mixed casing is an issue for you, **most adapters have a dedicated documentation section on how to force a casing convention.**

but no matter how much I look, I can't find the relevant documentation. Any idea where it is? (I use drizzle)

**Reply by adriantilvar** [reply]
Right there with you, bro. Have you found anything about it?

**Reply by Przeblysk** [reply]
```typescript
export const { handlers, signIn, signOut, auth } = NextAuth({
  adapter: DrizzleAdapter(db, {
    usersTable: users,
    accountsTable: accounts,
    sessionsTable: sessions,
    verificationTokensTable: verificationTokens
  }),
  providers: [GitHub]
})
```
Specify your schema in the Adapter
